chromium_src/chrome/browser/ui/views/infobars/confirm_infobar.cc

```diff
@@ -0,0 +1,9 @@
+#include "chrome/browser/infobars/infobar_service.h"
+
+#include "components/infobars/core/confirm_infobar_delegate.h"
+#include "components/infobars/core/infobar.h"
+
+std::unique_ptr<infobars::InfoBar> InfoBarService::CreateConfirmInfoBar(
+    std::unique_ptr<ConfirmInfoBarDelegate> delegate) {
+  delegate->Cancel();
```

Comment:
we don't have to return `base::MakeUnique<ConfirmInfoBar>(std::move(delegate))`?